docs/doxygen-user/volatility_dsp.dox

```diff
@@ -0,0 +1,32 @@
+/*! \page volatility_dsp_page Volatility Data Source Processor
+
+\section Overview
+
+The Volatility data source processor runs Volatility on a memory image and saves the individual module results. If the disk image associated with the memory image is also available, it will create Intesting Item artifacts linking the Volatility results to files in the disk image.
+
+\section Usage
+
+If you have a disk image associated with your memory image, ingest the disk image into the case first. Then go to "Add Data Source" and select "Memory Image File". 
+
```

Comment:
Does the user have to ingest a corresponding disk image? From what follows below, it sounds like they will have critical errors if they don't, but I read this sub section to mean the ingest of the disk image is optional...

Reply:
They're non-critical errors that appear at the end of the data source wizard. Essentially if they don't have the image they'll get the Volatility module reports but nothing else.

Reply:
Sorry, I misread "non-critical" as "critical." Hopefully, my new glasses will be ready soon. ;-)

Let's go ahead and mention the consequences of not ingesting the data source first, please.